.jenkins/pipelines/code-coverage.Jenkinsfile

```diff
@@ -5,79 +5,325 @@
 
 pipeline {
     agent {
-        label 'Jenkins-Shared-DC2'
+        label 'ACC-v3-2004'
     }
     options {
-        timeout(time: 300, unit: 'MINUTES')
+        timeout(time: 600, unit: 'MINUTES')
         timestamps ()
     }
     parameters {
         string(name: "REPOSITORY", defaultValue: "deislabs/mystikos")
         string(name: "BRANCH", defaultValue: "main", description: "Branch to build")
         choice(name: "REGION", choices:['useast', 'canadacentral'], description: "Azure region for the SQL solutions test")
         string(name: "PULL_REQUEST_ID", defaultValue: "", description: "If you are building a pull request, enter the pull request ID number here. (ex. 789)")
+        string(name: "COMMIT_SYNC", defaultValue: "", description: "optional - used to sync outputs of parallel jobs")
```

Comment:
Nit: this parameter is only needed when running sub jobs

Reply:
Removed this param